src/esrs/aggregate.rs

```diff
@@ -85,57 +84,32 @@ pub trait AggregateManager: Aggregate {
         let events: Vec<Self::Event> = <Self as Aggregate>::handle_command(aggregate_state.inner(), command)?;
         let stored_events: Vec<StoreEvent<Self::Event>> = self.store_events(&mut aggregate_state, events).await?;
 
-        Ok(<Self as AggregateManager>::apply_events(aggregate_state, stored_events))
-    }
-
-    /// Responsible for applying events in order onto the aggregate state, and incrementing the sequence number.
-    ///
-    /// `events` will be passed in order of ascending sequence number.
-    ///
-    /// You should _avoid_ implementing this function, and be _very_ careful if you decide to do so.
-    fn apply_events(
-        aggregate_state: AggregateState<Self::State>,
-        store_events: Vec<StoreEvent<Self::Event>>,
-    ) -> AggregateState<Self::State> {
```

Comment:
Removed for multiple reasons:
1. I'd argue overriding this function is never correct (the sequence number must be incremented in the proper way, and we must update the state as dictated by Aggregate). Sure, you can have your event store that doesn't check for sequence numbers and you can put garbage in your state, but why!?
2. more importantly, at the end of the function we were constructing the new AggregateState directly: but its fields were marked as `pub(crate)` so a consumer downstream could not even override this function (they wouldn't be able to create the return value!)